package-lock.json

```diff
@@ -0,0 +1,14761 @@
+{
```

Comment:
what do you think about ignoring `package-lock.json` and `yarn.lock` in the `.gitignore`. It seems like it may not make sense to commit these, since it won't matter for runtime users of the addon.